### PR TITLE
Update table view design

### PR DESCRIPTION
**Pull request by frederoni**
Outlined the structure of the new table view

- [x] Hook up voice, satellite & traffic.
- [ ] Add steps

@bsudekum @1ec5 👀

### CODE REVIEW

**Comment by frederoni** [comment]
@bsudekum Feel free to hook up the actions for satellite, traffic and sound.

File: MapboxNavigation/RouteTableViewController.swift [hunk 1]
```diff
@@ -13,43 +12,91 @@ class RouteTableViewController: UIViewController {
...
     
     func setupTableView() {
         tableView.tableHeaderView = headerView
-        tableView.rowHeight = UITableViewAutomaticDimension
-        tableView.estimatedRowHeight = 80
+        // TODO: Are we gonna use a progress bar?
+        //headerView.progress = CGFloat(routeController.routeProgress.fractionTraveled)
+        
+        let satellite = TableViewItem("Satellite")
+        let traffic = TableViewItem("Live traffic")
```
Comment (1ec5) [inline MapboxNavigation/RouteTableViewController.swift]: Use title case.

File: MapboxNavigation/RouteTableViewController.swift [hunk 2]
```diff
@@ -13,43 +12,91 @@ class RouteTableViewController: UIViewController {
...
     }
     
     func setupTableView() {
         tableView.tableHeaderView = headerView
-        tableView.rowHeight = UITableViewAutomaticDimension
-        tableView.estimatedRowHeight = 80
+        // TODO: Are we gonna use a progress bar?
+        //headerView.progress = CGFloat(routeController.routeProgress.fractionTraveled)
+        
+        let satellite = TableViewItem("Satellite")
```
Comment (1ec5) [inline MapboxNavigation/RouteTableViewController.swift]: These strings should be localizable.

**Comment by bsudekum** [comment]
We're holding on this for now.

**Comment by ericrwolfe** [comment]
Moved to the `future` board and closing for now.

**Pull request closed without merging**